src/main/java/com/autotune/analyzer/k8sObjects/ValidateAutotuneObject.java

```diff
@@ -68,6 +68,11 @@ public static StringBuilder validate(HashMap<String, Object> map) {
 			errorString.append(AnalyzerErrorConstants.AutotuneObjectErrors.DIRECTION_NOT_SUPPORTED);
 		}
 
+		//check if slo_class is 'response_time' and direction is minimize
+		if(sloInfo.getSloClass().equalsIgnoreCase("response_time") && sloInfo.getDirection().equalsIgnoreCase("maximize")){
```

Comment:
Please make sure spaces after `if` and before bracket `{`
```
if (...) {
}
```
I'd prefer if we check for `response_time` and `!minimize` throw error. Similarly, `throughput` and `!maximize` throw error

Reply:
Updated it now.

Reply:
I still dont see a space after if, `if (`

Reply:
Updated space between 'if' and brackets